recipes/harfbuzz/all/conanfile.py

```diff
@@ -43,20 +58,8 @@ class HarfbuzzConan(ConanFile):
 
     short_paths = True
 
-    generators = "cmake", "cmake_find_package"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
     def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
```

Comment:
In the `configure` method, can you use `rm_safe` when deleting the `fPIC` option? You'll need to bump the minimum Conan version to 1.53, too.